src/Pos/Util/BackupPhrase.hs

```diff
@@ -34,15 +34,17 @@ newtype BackupPhrase = BackupPhrase
 backupPhraseWordsNum :: Int
 backupPhraseWordsNum = 12
 
--- | Number of hashing rounds to get seed from phrase
-hashingRoundsNum :: Int
-hashingRoundsNum = 10000
+-- | Make backup phrase from list
+mkBackupPhrase12 :: [Text] -> BackupPhrase
```

Comment:
In fact, 12 is default. I got confused with 9 that we used to decrypt the certificate. We use 12 in daedalus and 9 is not used so far. Will remove